.github/workflows/ci.yml

```diff
@@ -42,49 +42,43 @@ jobs:
     env:
       GOPRIVATE: github.com/couchbaselabs
     steps:
-      - name: Setup Go Faster
-        uses: WillAbides/setup-go-faster@v1.8.0
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
         with:
           go-version: 1.21.1
-      - uses: actions/checkout@v2
       - name: Build
         run: go build -v "./..."
       - name: Run Tests
         run: go test -timeout=30m -count=1 -json -v "./..." | tee test.json | jq -s -jr 'sort_by(.Package,.Time) | .[].Output | select (. != null )'
         shell: bash
       - name: Annotate Failures
         if: always()
-        uses: guyarb/golang-test-annotations@v0.6.0
+        uses: guyarb/golang-test-annotations@v0.7.0
         with:
           test-results: test.json
 
   golangci:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: WillAbides/setup-go-faster@v1.8.0
-        with:
-          go-version: 1.21.1
       - uses: actions/checkout@v3
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.52.0
+          version: v1.54.2
 
   test-race:
     runs-on: ubuntu-latest
-    env:
-      GOPRIVATE: github.com/couchbaselabs
```

Comment:
I guess this isn't needed because we don't have any private dependencies. Was this just copypasta'd from SG's Actions?

Should we remove the one further up in this file also?

Reply:
yeah, just copypasta, removed.